debezium-core/src/test/java/io/debezium/junit/ConditionalFail.java

```diff
@@ -55,4 +72,37 @@ else if (failure != null) {
             throw new IllegalStateException(e);
         }
     }
+
+    private Statement ignoreFlakyFailure(final Statement base, final Description description,
+                                         final Flaky flakyClass) {
+
+        final String flakyFailuresProperty = System.getProperty(Flaky.IGNORE_FLAKY_FAILURES_PROPERTY);
```

Comment:
According to the renaming proposed in my other 2 nit-pick comments I would rename those consts accordingly.